var/spack/repos/builtin/packages/python/package.py

```diff
@@ -203,8 +203,11 @@ def python_ignore(self, ext_pkg, args):
                 r'site-packages/site[^/]*\.pyc?$',
                 r'site-packages/__pycache__/site[^/]*\.pyc?$'
             ])
+        if ext_pkg.name != 'py-pygments':
+            patterns.append(r'bin/pygmentize$')
```

Comment:
This affected one package, but I can imagine this coming up in others as well, so it may make sense to leave this in the Python package.